public/hyperswitch/index.html

```diff
@@ -1,108 +1,132 @@
+      -webkit-font-smoothing: antialiased;
+      padding: 0;
+      margin: 0;
+      height: 100%;
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+    }
+
+    #app {
+      display: flex;
+      flex-direction: column;
+      flex: 1;
+    }
+
+    .list-item {
+      background-color: rgba(255, 255, 255, 1);
+      padding: 10px;
+      transition: background-color 0.2s;
+    }
 
-      .linkAnnotation a {
-        display: flex;
-        width: 100%;
-        height: 100%;
+    .list-item:hover {
+      background-color: rgba(0, 0, 0, 0.1);
+    }
+
+    .clickable {
+      cursor: pointer;
+    }
+
+    .react-pdf__Page__annotations {
+      height: auto !important;
+      width: 0 !important;
+    }
+
+    .linkAnnotation {
+      width: 20% !important;
+    }
+
+    .linkAnnotation {
+      position: absolute;
+    }
+
+    .linkAnnotation a {
+      display: flex;
+      width: 100%;
+      height: 100%;
+    }
+
+    @media (prefers-color-scheme: dark) {
+      body {
+        color: rgba(229, 231, 235, 1);
+        background-color: rgba(15, 16, 17, 1);
       }
+    }
+  </style>
+  <link rel="stylesheet" href="/app.css" />
 
-      @media (prefers-color-scheme: dark) {
-        body {
-          color: rgba(229, 231, 235, 1);
-          background-color: rgba(15, 16, 17, 1);
+  <script>
+    function clearCookiesWithURLs() {
+      document.cookie.split(";").forEach(function (c) {
+        c = c.trim();
+        cookie_name = c.split("=")[0];
+        cookie_value = c.split("=")[1];
+        cookie_value = decodeURIComponent(cookie_value);
+        if (
+          cookie_value.indexOf("https://") > -1 ||
+          cookie_value.indexOf("http://") > -1
+        ) {
+          document.cookie =
+            cookie_name + "=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;";
         }
-      }
-    </style>
-    <link rel="stylesheet" href="/app.css" />
-
-    <script>
-      function clearCookiesWithURLs() {
-        document.cookie.split(";").forEach(function (c) {
-          c = c.trim();
-          cookie_name = c.split("=")[0];
-          cookie_value = c.split("=")[1];
-          cookie_value = decodeURIComponent(cookie_value);
-          if (
-            cookie_value.indexOf("https://") > -1 ||
-            cookie_value.indexOf("http://") > -1
-          ) {
-            document.cookie =
-              cookie_name + "=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;";
-          }
-        });
-      }
-    </script>
-  </head>
-
-  <body>
-    <!-- SDK Loader Manager (noscript) -->
-    <script type="module" src="/wasm/euclid.js"></script>
-    <script src="/module.js"></script>
-    <div id="app"></div>
-    <script onload="clearCookiesWithURLs()" src="/app.js"></script>
-  </body>
-</html>
+      });
+    }
+  </script>
+</head>
+
+<body>
+  <!-- SDK Loader Manager (noscript) -->
+  <script defer type="module" src="/wasm/euclid.js"></script>
+  <script defer src="/module.js"></script>
```

Comment:
For both of this can use async

Reply:
Can we also move these scripts below `app.js`

Reply:
not required

Reply:
Let it be defer only as async may cause issues. In defer let the script fully load and then execution start

Reply:
These are scripts which are not needed from the start and also does not need DOM to render hence we can use async